### ISSUE TITLE
Feature request: Custom trigger

### ISSUE THREAD

**Issue by DRFR0ST** [arfedulov/semantic-ui-calendar-react#52]
**Custom trigger**

Please implement a custom trigger (eg. Button) instead of the default input. It could be working like the [Popup](https://react.semantic-ui.com/modules/popup/#types-popup) from Semantic-UI but instead of providing the `content` prop, it would contain one of the pickers.

Here is an example made using the Popup component and inline TimeInput.
![screenshot from 2018-12-05 12-28-32](https://user-images.githubusercontent.com/7634871/49510986-24b7ca00-f88a-11e8-812a-37699c2377d2.png)

It would be used like that
`<TimeInput trigger={<Button>Pick!</Button>} ... />`

I might find some time to create that feature and make a pull request but first i would like to know what you think about a feature like that.

**Comment by arfedulov** [comment]
Yeah, let's give it a try. But I think that it is worth to consider implementing it as additional top level `*Input` component. Because component's interface is getting cluttered and there are many props that couldn't be used together.
